sympy/sets/sets.py

```diff
@@ -206,6 +206,12 @@ def _complement(self, other):
         elif isinstance(other, FiniteSet):
             return FiniteSet(*[el for el in other if self.contains(el) != True])
 
+    def _symmetricdifference(self, other):
```

Comment:
Probably, we should have a pair of private and public methods, just like other cases.  Also, please see documentation of [builtin set](https://docs.python.org/3/library/stdtypes.html#set-types-set-frozenset) class.  I think, that's why public method should be named as `symmetric_difference`.